examples/tutorials/ProteinLM_Tutorial0.ipynb

```diff
@@ -0,0 +1,2855 @@
+{
```

Comment:
<p>Formatting as mentioned above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4054/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I also see this table properly formatted</p>